examples/graphql-service-error-handling/graphql_service_error_handling.md

```diff
@@ -0,0 +1,35 @@
+# GraphQL service - Error handling
+
+The Ballerina `graphql` module allows returning `error`s from the `resource` and `remote` methods used as GraphQL object fields. If a field returns an error, it will be added in the GraphQL response under the `errors` field, while the field value is set to `null`. In this case, if the GraphQL field type is `NON_NULL`, the `null` value is propagated to the upper levels, until a `null` value is allowed as the field value. This might cause the whole `data` field to be `null` in some cases. Alternatively, the field type can include nil, so the field value can be `null`. In that case, the `null` value is not propagated to the upper levels in the response.
+
+::: code graphql_service_error_handling.bal :::
+
+Run the service by executing the following command.
```

Comment:
```suggestion
Run the service by executing the following command.
```
Shall we add these as numbered steps?

Reply:
We did not add numbers to the BBEs similar to this. Wouldn't adding numbers make this out-of-convention?

Reply:
The best practice in technical documentation is to add sequential instructions as numbered bullet points. We can adhere to this as now we have the MD format.

Reply:
In that case, we might have to do this in every BBE. @shafreenAnfar WDYT?

Reply:
Lets not change the convention now. Can consider it for a later revamp.